packages/theming/theme-tokens/src/light-macos/reactnative/tokens-aliases.json

```diff
@@ -1,262 +1,262 @@
-	}
-}
\ No newline at end of file
+  "body": {
+    "fontFamily": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"",
+    "fontLineHeight": 20,
+    "fontSize": 14,
+    "fontWeight": "400"
+  },
+  "bodyStrong": {
+    "fontFamily": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"",
+    "fontLineHeight": 20,
+    "fontSize": 14,
+    "fontWeight": "600"
+  },
+  "borderless": {
+    "strokeAlignment": "inner",
+    "strokeColorRest": "transparent",
+    "strokeColorHover": "transparent",
+    "strokeColorPressed": "transparent",
+    "strokeColorDisabled": "transparent",
+    "strokeWidth": 1
+  },
+  "brandBackground": {
+    "fillColorRest": "#185abd",
+    "fillColorHover": "#1651aa",
+    "fillColorPressed": "#0e336a",
+    "fillColorSelected": "#13458f"
+  },
+  "brandBackground2": {
+    "fillColorRest": "#d2e0f4"
+  },
+  "brandBackgroundStatic": {
+    "fillColorRest": "#185abd"
+  },
+  "brandForeground1": {
+    "fillColorRest": "#185abd"
+  },
+  "brandForeground2": {
+    "fillColorRest": "#1651aa"
+  },
+  "brandForegroundLink": {
+    "fillColorRest": "#185abd",
+    "fillColorHover": "#1651aa",
+    "fillColorPressed": "#0e336a",
+    "fillColorSelected": "#13458f"
+  },
+  "brandStroke1": {
+    "strokeColorRest": "#185abd",
+    "strokeWidth": 1
+  },
+  "brandStroke2": {
+    "strokeColorRest": "#6794d7",
+    "strokeWidth": 1
+  },
+  "caption": {
+    "fontFamily": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"",
+    "fontLineHeight": 16,
+    "fontSize": 12,
+    "fontWeight": "400"
+  },
+  "compoundBrandBackground1": {
+    "fillColorRest": "#185abd",
+    "fillColorHover": "#1651aa",
+    "fillColorPressed": "#0e336a"
+  },
+  "compoundBrandForeground1": {
+    "fillColorRest": "#185abd",
+    "fillColorHover": "#1651aa",
+    "fillColorPressed": "#0e336a"
+  },
+  "compoundBrandStroke1": {
+    "strokeColorRest": "#185abd",
+    "strokeColorHover": "#1651aa",
+    "strokeColorPressed": "#0e336a",
+    "strokeWidth": 1
+  },
+  "display": {
+    "fontFamily": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"",
+    "fontLineHeight": 92,
+    "fontSize": 68,
+    "fontWeight": "600"
+  },
+  "headline": {
+    "fontFamily": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"",
+    "fontLineHeight": 28,
+    "fontSize": 20,
+    "fontWeight": "600"
+  },
+  "largeTitle": {
+    "fontFamily": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"",
+    "fontLineHeight": 52,
+    "fontSize": 40,
+    "fontWeight": "600"
+  },
+  "neutralBackground1": {
+    "fillColorRest": "#ffffff",
+    "fillColorHover": "#f5f5f5",
+    "fillColorPressed": "#e0e0e0",
+    "fillColorSelected": "#ebebeb"
+  },
+  "neutralBackground2": {
+    "fillColorRest": "#fafafa",
+    "fillColorHover": "#f0f0f0",
+    "fillColorPressed": "#dbdbdb",
+    "fillColorSelected": "#e6e6e6"
+  },
+  "neutralBackground3": {
+    "fillColorRest": "#0000000C",
```

Comment:
first time seeing this hex but ok

Reply:
Actually that's a good point: I don't see this value in the global tokens; seems like it should be `#F5F5F5`. Any idea what happened here @chiuam @rurikoaraki?

Reply:
@chiuam any ideas? It doesn't look like the input file was touched, so I'm not sure why the output files were changed

Reply:
I copied the value from the apple theme because it looked a lot closer to the existing button, here's a visual comparison: 
| #F5F5F5                                       | #0000000C                                      |
|----------|--------|
| ![image](https://user-images.githubusercontent.com/67026167/144653287-742cb60d-d4f9-492f-b718-bab6dfed0aaa.png) | ![image](https://user-images.githubusercontent.com/67026167/144653326-91a92e71-fbf7-4af2-8a43-b061583a5e8f.png) |

Reply:
I'm not sure whether or not we should go with `#F5F5F5` for buttonBackground, though I do think the hint of grey looks more natural to me. @Saadnajmi thoughts?

Reply:
I think we should follow the [spec](https://www.figma.com/file/ArWQiU7BAlrDGNmbrGEXbA/%5BSpecs%5D-Fluent-vNext-iOS-Components?node-id=744%3A734) for now to move forward and revisit the button background colors when the mac tokens are more finalized.  I'll revert the changes I made to the token values.

Reply:
I copied the apple theme background button color from Fluent UI Apple. I would vote we open a bug on the design token repo or something to fix the macOS token, and follow spec. Especially if both options look acceptable in production, which it seems to be